README.md

```diff
@@ -60,174 +28,112 @@ After you're finished please remove all the comments and instructions!
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
+# 📖 [Abhii's Portfolio] <a name="about-project"></a>
 
-> Describe your project in 1 or 2 sentences.
-
-**[your_project__name]** is a...
+> It is second project of microverse building a portfolio.
 
+**Abhii's Portfolio** is a great project to learn HTML and CSS.
 ## 🛠 Built With <a name="built-with"></a>
 
 ### Tech Stack <a name="tech-stack"></a>
 
-> Describe the tech stack and include only the relevant sections that apply to your project.
+> This project is compiled using HTML, CSS and JS.
 
 <details>
   <summary>Client</summary>
   <ul>
-    <li><a href="https://reactjs.org/">React.js</a></li>
+    <li><a href="https://Javascript.org/">Javascript</a></li>
   </ul>
 </details>
 
 <details>
   <summary>Server</summary>
   <ul>
-    <li><a href="https://expressjs.com/">Express.js</a></li>
+    <li><a href="https://apache.com/">Apache</a></li>
   </ul>
 </details>
 
 <details>
 <summary>Database</summary>
   <ul>
-    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+    <li><a href="https://www.postgresql.org/">None</a></li>
   </ul>
```

Comment:
Currently, the text `None` is directed to the Postgres website `href="https://www.postgresql.org`.

It would be nice if we remove the `link` around the text for a professional readme. or we remove the entire `Database` section since it is out of the scope of this particular project ☺


```
<details>
<summary>Database</summary>
  <ul>
    <li><a href="https://www.postgresql.org/">None</a></li>
  </ul>
</details>
```